Storage/rest/types.h

```diff
@@ -234,6 +234,44 @@ CURRENT_STRUCT_T(HypermediaRESTCollectionResponse) {
   CURRENT_FIELD(data, std::vector<T>);
 };
 
+// The simplest way to pass universal dictionary/matrix key and entry to the templated Current struct.
+template <typename KEY, typename ENTRY>
+struct DetailedExportEntryHelper {
+  using key_t = KEY;
+  using entry_t = const ENTRY&;
```

Comment:
$0.02: This does look a bit confusing, especially in the signature of the constructor.